modules/Sources/Features/SendFlow/SendFlowStore.swift

```diff
@@ -90,7 +91,10 @@ public struct SendFlowReducer: Reducer {
         }
         
         public var isMemoInputEnabled: Bool {
-            transactionAddressInputState.textFieldState.text.data.isEmpty ||
+            print(transactionAddressInputState.textFieldState.text.data.isEmpty ||
```

Comment:
Is this a debug print that is inadvertently left in place?

Reply:
Yes, debug, removed.